sdk/communication/communication-email/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.0.1-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1-beta.2 (2024-12-20)
```

Comment:
If you're releasing, you will need to set the correct date. Also why is it v1.0.1-beta.2 and not v1.0.0-beta.2? We typically don't release beta batch releases.

```suggestion
## 1.0.0-beta.2 (2024-12-23)
```

Reply:
I went ahead and updated to today's date. As for the versioning, there's a couple reasons. For one, we already released a 1.0.0-beta.2 previously. The other reason is that in our last SDK update, we were advised by the SDK team to update from our GA version (1.0.0) to 1.0.1-beta.1. Seeing as this is a patch/fix to add add'l content to the README, it seemed to make sense in this case to keep the numbering the same but increment the beta patch number.